store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -137,6 +141,31 @@ public void initialise(final Schema schema, final StoreProperties properties)
         TableUtils.ensureTableExists(this);
     }
 
+    @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+
+        final ValidationResult validationResult = new ValidationResult();
+
+        getSchemaElements().entrySet().forEach(entrySet -> entrySet.getValue().getProperties().forEach(propertyName -> {
```

Comment:
As a large number of properties may use the same 'type' so you might end up checking the same type serialiser multiple times.

Rather than iterating around all the properties it would be more efficient to iterate around the schema 'types'.